storybookjs/storybook
# Add font family to info panel

Issue: #5723 

## What I did

Add the same font family to the info panel that is used by the outer panels.

## How to test

Use the info addon. I tested with the cra-kitchen-sink example. Previously, the info panel did not have the same font family as the outer panel(s).

- Is this testable with Jest or Chromatic screenshots? Possibly
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
